GUI/src/main/kotlin/ui/screens/SelectVideoScreen.kt

```diff
@@ -70,3 +101,28 @@ fun SelectVideoScreen(state: MutableState<AppState>) {
         })
     }
 }
+
+private fun checkVideoFormatAndCodec(
```

Comment:
Could we move the responsibility to check this to lib2? Like, move this function to some `CodecChecker` class in lib2, which both the gui and lib2 use? This would prevent later confusion when adapting allowed codecs :)

Reply:
We could also extend the AcceptedCodecs Class in lib 2 to include the currently used codec and then access that in the gui. Idea being to notice a wrong codec at the file selection process

Reply:
That would be good, I think. Can you manage to do that til the sprint end?